build.gradle

```diff
@@ -191,9 +191,9 @@ dependencies {
     compile group: 'com.microsoft.azure', name: 'applicationinsights-logging-log4j2', version: '2.4.0'
 
     testCompile 'junit:junit:4.12'
```

Comment:
I guess this can removed now?

Reply:
You can try it, but we are still using some of the old junit4 parameterized tests